.travis.yml

```diff
@@ -1,32 +1,15 @@
 language: php
 
 php:
+    - 7.3
+    - 7.2
     - 7.1
-    - 7.0
-    - 5.6
     - nightly
 
 matrix:
     fast_finish: true
-    include:
-        - php: 5.6
-          env: COMPOSER_FLAGS="--prefer-lowest"
-        - php: 5.6
-          env: SYMFONY_VERSION=2.7.*
-        - php: 5.6
-          env: SYMFONY_VERSION=2.8.*
-        - php: 5.6
-          env: SYMFONY_VERSION=3.1.*
-        - php: 5.6
-          env: SYMFONY_VERSION=3.2.*
-        - php: 5.6
-          env: SYMFONY_VERSION=3.3.*
-        - php: 5.6
-          env: SYMFONY_VERSION=3.4.*@dev
```

Comment:
It would be nice to keep a matrix for supported symfony versions, something like : 
```yaml
matrix:
    fast_finish: true
    include:
        - php: 7.1
          env: COMPOSER_FLAGS="--prefer-lowest"
        - php: 7.1
          env: SYMFONY_VERSION=2.8.*
        - php: 7.1
          env: SYMFONY_VERSION=3.4.*
        - php: 7.1
          env: SYMFONY_VERSION=4.3.*
        - php: 7.1
          env: SYMFONY_VERSION=4.4.*@dev
    allow_failures:
        - php: nightly
        - php: 7.1
          env: SYMFONY_VERSION=4.4.*@dev
```

Reply:
Oh, I'm sorry, I was not so familiar with this matrix. But you are right that all supported versions should be tested.